sova/awareness/providers/gcal.py

```
@@ -0,0 +1,274 @@
+            creds = authenticate_google(self.config)
+        except Exception:
+            _log.warning("gcal.auth_failed", exc_info=True)
+            return []
+
+        try:
+            service = build("calendar", "v3", credentials=creds)
+        except Exception:
+            _log.warning("gcal.service_build_failed", exc_info=True)
+            return []
+
+        now = datetime.now()
```

Ruff (0.16.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)